Resources/Private/Partials/Columns/Inline.html

```
@@ -0,0 +1,52 @@
+    xmlns:core="http://typo3.org/ns/TYPO3/CMS/Core/ViewHelpers"
+    xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers">
+
+<f:be.pageRenderer includeJavaScriptModules="{0: '@xima/recordlist/recordlist-badge-limit.js'}" />
+
+<span class="badge-container inline-badge-container" data-column-name="{colConfig.columnName}">
+    <f:if condition="{item._{colConfig.columnName}}">
+        <f:for each="{item._{colConfig.columnName}}" as="relations" key="tableName">
+            <f:for each="{relations}" as="relation" iteration="i">
+                <f:if condition="{item.editable}">
+                    <f:then>
+                        <xm:link.editRecord
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)